docs/setup.md

```diff
@@ -11,6 +11,7 @@ cargo install gitql
 ## Winget on Windows
 
 ```sh
+winget install cmake
```

Comment:
Thank you, but The cmake is required only if you want to build it from source code, but not required if you install executable from any package manager

Reply:
Sorry for that.
https://github.com/AmrDeveloper/GQL/raw/master/media/gql_demo.gif   in https://github.com/AmrDeveloper/GQL/blob/master/README.md   is missing the key step "install cmake first"
Maybe update the gif would be a good idea.